packages/api/src/utils/client/httpStatusCode.ts

```diff
@@ -0,0 +1,394 @@
+"use strict";
+
+/**
+ * Hypertext Transfer Protocol (HTTP) response status codes.
+ * @see {@link https://www.rfc-editor.org/rfc/rfc7231#section-6}
+ */
+export enum HttpStatusCode {
```

Comment:
Who is using all these protocols? What's the point of declaring this?

Reply:
To create types for the API which is completely matching the specification.